pkg/systemd/logs.jsx

```diff
@@ -273,6 +274,7 @@ export const LogsPage = () => {
                          variant={PageSectionVariants.light}
                          id="journal-box">
                 <JournalBox dataFollowing={dataFollowing}
+                            defaultSince={timeFilter.value}
```

Comment:
This must be where the [tests crash](https://logs.cockpit-project.org/logs/pull-16349-20210914-141315-c1a05c4f-fedora-35/log.html#240-2):

    RuntimeError: TypeError: Cannot read property 'value' of undefined

Reply:
Ah, that's why I initially thought about passing `timeFilter`, will fix.

Reply:
It's due to the test giving since=@123123213 which doesn't match anything and returns undefined and I was somehow thinking the code would always fallback to my default else.